### PR TITLE
Feature/ari 173

### PR DESCRIPTION
**Pull request by alexishecf**

### CODE REVIEW

File: packages/service-provider/src/lib/service-provider.ts [hunk 1]
```diff
@@ -264,6 +264,8 @@ export class ServiceProvider {
       protocolSlug
     );
 
+    console.log(owner, tokenId, to);
```
Comment (CyrilCartoux) [inline packages/service-provider/src/lib/service-provider.ts:267]: to be removed ?

File: packages/token-provider/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# token-provider
```
Comment (CyrilCartoux) [inline packages/token-provider/README.md:1]: Can we get more infos on this library ? 
What is it ? How it works? What it is used for ? 😇

File: packages/token-provider/src/lib/token-provider.ts [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    permitTransferFrom,
+    permit721Address,
+    smartAsset.protocol.chainId
+  );
+
+  const permitSignature = (
+    await core.signTypedData!(tddAdapter(domain), types, values)
+  ).signature;
+
+  const sst = aatInstance.createCertificateArianeeAccessToken(
```
Comment (CyrilCartoux) [inline packages/token-provider/src/lib/token-provider.ts:61]: missing await

**Review by CyrilCartoux — DISMISSED** [review]

**Review by CyrilCartoux — APPROVED** [review]

**Review by 0xWhiteleaf — APPROVED** [review]

**Comment by 0xWhiteleaf** [comment]
ready for merge

**Pull request merged**